File: leetcode/palindromeNumber/solution1.c
bool isPalindrome(int x) {
    int y = x;
    double num = 0;
    while (x > 0) {
        num = (num*10) + (x%10);
        x = x/10;
    }
    if(y == num) {
        return true;
    } else {
        return false;
    }
}

File: leetcode/validPalindrome/solution1.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function(s) {
    s = s.toLowerCase().replace(/[^a-z0-9]/g, '');
    s2 = '';

    for(let i = s.length - 1; i >= 0; i--) {
        s2 = s2 + s[i];
    }

    if(s === s2) {
        return true;
    } else {
        return false;
    }
};